src/main/java/appeng/fluids/container/ContainerFluidInterface.java

```diff
@@ -67,56 +67,49 @@ public void detectAndSendChanges()
 
 		if( Platform.isServer() )
 		{
-			this.sendTankUpdate();
+			this.tankSync.sendDiff( this.listeners );
 		}
 
 		super.detectAndSendChanges();
 	}
 
 	@Override
-	public void addListener( IContainerListener listener )
+	protected void setupConfig()
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1186)